### PR TITLE
ready for review

### PR DESCRIPTION
**Pull request by skeydan**

### CODE REVIEW

File: _posts/2020-07-20-fnn-lstm/timeseries-prediction-with-fnn-lstm.Rmd [hunk 1]
```diff
@@ -0,0 +1,829 @@
...
+Keeping up the suspense for a while, here is the code used to compute per-timestep prediction errors from both models. The
+same code will be used for all other datasets.
+
+```{r, eval = FALSE}
+calc_mse <- function(df, y_true, y_pred) {
+  (sum((df[[y_true]] - df[[y_pred]])^2))/nrow(df)
+}
+
+get_mse <- function(test_batch, prediction) {
+  comp_df <- data.frame(test_batch[[2]][, , 1] %>%
```
Comment (dfalbel) [inline _posts/2020-07-20-fnn-lstm/timeseries-prediction-with-fnn-lstm.Rmd:571]: This function is hard to parse, I think splitting the pipe blocks would make it easier.

File: _posts/2020-07-20-fnn-lstm/timeseries-prediction-with-fnn-lstm.Rmd [hunk 2]
```diff
@@ -0,0 +1,829 @@
...
+lstm <- data.frame(as.array(prediction_lstm[, , 1]) %>%
+                     t()) %>%
+  add_column(type = "lstm") %>%
+  add_column(num = (n_timesteps + 1):(2 * n_timesteps))
+
+compare_preds_df <- bind_rows(given, lstm, fnn)
+
+plots <- purrr::map(sample(1:dim(compare_preds_df)[2], 16),
+                    function(v)
+                      ggplot(compare_preds_df, aes(num, .data[[paste0("X", v)]], color = type)) +
```
Comment (dfalbel) [inline _posts/2020-07-20-fnn-lstm/timeseries-prediction-with-fnn-lstm.Rmd:641]: I think adding the `{` for functions makes it easier to read.

File: _posts/2020-07-20-fnn-lstm/timeseries-prediction-with-fnn-lstm.Rmd (hunk 1 above)
Comment (skeydan) [reply _posts/2020-07-20-fnn-lstm/timeseries-prediction-with-fnn-lstm.Rmd:571]: Right! Think I found a way to make the structure more clear now :-)

File: _posts/2020-07-20-fnn-lstm/timeseries-prediction-with-fnn-lstm.Rmd (hunk 2 above)
Comment (skeydan) [reply _posts/2020-07-20-fnn-lstm/timeseries-prediction-with-fnn-lstm.Rmd:641]: yeah, agree, will do!
